nautobot/extras/models/jobs.py

```diff
@@ -1159,15 +1171,16 @@ def create_schedule(
         return scheduled_job
 
     def to_cron(self):
-        t = self.start_time
+        t = self.start_time.astimezone(self.time_zone)
```

Comment:
This is a critical part of the bug fix, and is probably the root issue of #5591.